examples/lit-icd-app/linux/args.gni

```diff
@@ -28,6 +28,6 @@ matter_enable_tracing_support = true
 
 # ICD configurations
 chip_enable_icd_server = true
-chip_subscription_timeout_resumption = false
+chip_subscription_timeout_resumption = true
```

Comment:
Why is this needed?  Neither the issue, nor the PR description / commit message, nor comments in the gni (ideal place for it) explain.

Reply:
I will update the description. I was waiting on some validation. Putting this in draft for now